homeassistant/components/conversation/default_agent.py

```diff
@@ -623,6 +639,22 @@ def _make_slot_lists(self) -> dict[str, SlotList]:
 
         return self._slot_lists
 
+    def _make_intent_context(
```

Comment:
I think that custom intents would benefit quite a lot from the inclusion of a satellite `device_id`. That way, we can know exactly which satellite was used to trigger an intent.